modules/local/nf-core-modified/iqtree.nf

```diff
@@ -9,13 +9,13 @@ process IQTREE {
         'quay.io/biocontainers/iqtree:2.1.4_beta--hdcc8f71_0' }"
 
     input:
-    tuple val(meta), path(alignment)
```

Comment:
yeahhhh, probably good to just distance ourselves from using tuples like this whenever possible.